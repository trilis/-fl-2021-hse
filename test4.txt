01a0a010a10a10
