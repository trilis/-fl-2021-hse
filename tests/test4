Q : 0 1 
A : na nb  
( q0 , T , na , q1 , I ) ( q1 , I , na , q1 , I ) 
( q1 , I , nb , q0 , T )  
