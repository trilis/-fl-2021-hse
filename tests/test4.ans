: syntax error, line :7
