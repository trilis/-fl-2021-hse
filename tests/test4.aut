ALPH_SIZE=2
START_ST=1
ST_NUM=3 
ED_NUM=4
"0" "1"
STATE 1, lul 
STATE 2, lol 
STATE 3, kekas
EDGE 1, 2, "0"
EDGE 2, 1, "1"
EDGE 1, 2, "1"
EDGE 2, 3, "0"
