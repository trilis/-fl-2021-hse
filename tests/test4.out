Exception: final state not in set of states